src/views/Timeline.vue

```diff
@@ -45,53 +45,65 @@
           </UiButton>
         </UiDropdown>
       </div>
+
       <Block v-if="spaces.length < 1 && !scope" class="text-center">
         <div class="mb-3">{{ $t('noFavorites') }}</div>
         <router-link :to="{ name: 'home' }">
           <UiButton>{{ $t('addFavorites') }}</UiButton>
         </router-link>
       </Block>
+
       <Block v-else-if="loading" :slim="true">
         <RowLoading class="my-2" />
       </Block>
 
-      <NoResults
-        :block="true"
-        v-else-if="Object.keys(this.proposals).length < 1"
-      />
+      <NoResults :block="true" v-else-if="this.proposals.length < 1" />
       <div v-else>
         <Block :slim="true" v-for="(proposal, i) in proposals" :key="i">
           <TimelineProposal :proposal="proposal" :i="i" />
         </Block>
       </div>
+      <div id="endsensor"></div>
+      <Block v-if="loadingMore && !loading" :slim="true">
+        <RowLoading class="my-2" />
+      </Block>
     </template>
   </Layout>
 </template>
 
 <script>
 import { subgraphRequest } from '@snapshot-labs/snapshot.js/src/utils';
+import scrollMonitor from 'scrollmonitor';
+
+const loadBy = 15;
 
 export default {
   data() {
     return {
       loading: false,
-      proposals: {},
+      loadingMore: false,
+      stopLoadingMore: false,
+      proposals: [],
       scope: this.$route.params.scope,
       state: 'all',
-      spaces: []
+      spaces: [],
+      limit: loadBy
     };
   },
   watch: {
-    state() {
-      this.loadProposals();
+    async state() {
+      this.proposals = [];
+      this.limit = loadBy;
+      this.loading = true;
+      await this.loadProposals();
+      this.loading = false;
     }
   },
   methods: {
     selectState(e) {
       this.state = e;
     },
-    async loadProposals() {
-      this.loading = true;
+    async loadProposals(skip = 0) {
       this.spaces =
         this.scope === 'all' ? [] : Object.keys(this.favoriteSpaces.favorites);
```

Comment:
@bonustrack where is `this.favoriteSpaces` coming from?

Reply:
@samuveth It come from the store https://github.com/snapshot-labs/snapshot/blob/bccb285999428052fa95f105c2ba007ebc7c1335/src/store/modules/favoriteSpaces.ts

Reply:
How is it available like that in the `timeline.vue` component?

Reply:
@samuveth because of this: https://github.com/balancer-labs/snapshot/blob/8895bcef6ad400834e643f0d06b5193181b22b07/src/mixins.ts#L16